[MRG] Fix seeg coordinates to be in MNI space (#4)
mne-tools/mne-misc-data: README.md

# mne-misc-data
Miscellaneous data

## SEEG Example Data

> Note that the electrode coordinates are in subject's MRI space. The corresponding ``*talairach.xfm`` file provides the transformation to 
> get to ``fsaverage`` MNI space.

Reviewer: ```suggestion
Note that the electrode coordinates were transformed from the subject's
FreeSurfer surface RAS (MRI) space to MNI space.
```
